herald/herald/Sensor/SensorArray.swift

```diff
@@ -41,22 +41,6 @@ public class SensorArray : NSObject, Sensor {
             sensorArray.append(ConcreteInertiaSensor());
             add(delegate: CalibrationLog(filename: "calibration.csv"));
         }
-
-        if let payloadData = payloadData {
-            // Loggers
-            #if DEBUG
```

Comment:
Only the lines in the #if should have been moved out of sensorarray. The other ifs and logging are not related to these log sensors.